File: mario.c
#include <stdio.h>
#include <cs50.h>

int main (void){
    int h;
    int c = 0;
    int p = 0;
    
    do{
        h = get_int("height: \n");
        
    } while (!(h > 0 && h <= 8));
    
    
    for (int i = 1; i <= h; i++){
        c = h - i;
        p = i;
        while (c > 0){
           
            printf(" ");
            
            c--;
            
        }
        
        while (p > 0){
           
            printf("#");
            
            p--;
            
        }
        
        printf("\n");
    }
    
}

File: mario.py
import cs50 as cs

while True:
    height = cs.get_int("Height: ")
    
    if height >= 1 or height <= 8:
        break
i = 1
for i in range(height + 1):
    c = height - i
    p = i
    while c > 0:
        print(" ", end = "")
        c -= 1
        
    while p > 0:
        print("#", end = "")
        p -= 1
    
    print("\n", end = "")
